src/main/java/inventorycount/InventoryCountController.java

```diff
@@ -0,0 +1,31 @@
+package inventorycount;
+
+import javax.naming.InsufficientResourcesException;
+import java.util.HashMap;
+import java.util.UUID;
+
+public class InventoryCountController {
+
+    private final InventoryCountInteractor inventoryCountInteractor;
+
+    public InventoryCountController(UUID facID){
+
+        this.inventoryCountInteractor = new InventoryCountInteractor(facID);
+
+    }
+
+
+    public void submitCount(HashMap<Integer, Integer> count){
```

Comment:
just to clarify, the view will sending a hashmap to the controller? so the user will make some selections and changes via the view and then the view will handle the logic of turning that into a hashmap? i'm wondering if it would be better for the controller to handle that, but i'm not sure